.github/workflows/unit-tests.yml

```diff
@@ -29,9 +29,77 @@ jobs:
           pip install -e .[sbi]
           git clone https://github.com/florpi/ili-summarizer.git
           pip install -e ili-summarizer
+          pip install coverage pytest
       - name: Test with pytest
         run: |
-          pip install pytest pytest-cov
-          pytest --cov-report=xml --cov=ili tests/test_sbi.py
-          cat coverage.xml
+          echo "Running sbi tests"
+          COVERAGE_FILE=.coverage_sbi python3 -m coverage run --source=ili -m pytest tests/test_sbi.py
+          echo "Running util tests"
+          COVERAGE_FILE=.coverage_utils python3 -m coverage run --source=ili -m pytest tests/test_utils.py
+        shell: bash
+      - name: Archive sbi code coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage_sbi
+          path: .coverage_sbi
+      - name: Archive utils code coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage_utils
+          path: .coverage_utils
+        
+  pydelfi_build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.6"
+      - name: Setup MPI
+        uses: mpi4py/setup-mpi@v1
+        with:
+          mpi: 'openmpi'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          pip install -e .[pydelfi]
+          git clone https://github.com/florpi/ili-summarizer.git
```

Comment:
Remove?

Reply:
Lines 67-69 are still needed, but I have removed the ili-summarizer installation.